app/_src/policies/meshaccesslog.md

```diff
@@ -28,13 +28,20 @@ If you haven't, see the [observability docs](/docs/{{ page.release }}/explore/ob
 | `from[].targetRef.kind` | `Mesh`                                                   |
 {% endif_version %}
 {% endif_version %}
-{% if_version gte:2.9.x %}
+{% if_version eq:2.9.x %}
```

Comment:
Should we update https://github.com/kumahq/kuma-website/blob/master/app/_src/policies/targetref.md ?

Reply:
I will be updating this in next prs